news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+And before we finish, let me ask you some bonus questions, starting with the classic one: What plays, be they yours or someone else’s, you find the most impressive? If we pick one of yours, which one are you the most proud of?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+If I were to pick the most impressive score, it would be probably ["Oracion(TV-Size) [Crystal's Rain]" +HDDTHR](https://osu.ppy.sh/scores/fruits/173627724) played by [ExGon](https://osu.ppy.sh/users/214187), because high approach rate (mostly AR10.5~11) in osu!catch is really tricky and difficult skillset to master.
+
+Also there are a few candidates that are coming to my mind, but if I have to pick one of them, it will be ["Sendan Life (katagiri Bootleg) [World End.]"](https://osu.ppy.sh/scores/fruits/214502143) without a doubt, because this map literally requires most of skillsets which are "consistency", "speed", "stamina" and also to quote ExGon, [“probably the best nomod score for ctb maps, no one will be able to get close for a while”](https://twitter.com/ExG0n/status/1713871203314868578). So I do believe this is enough to be picked.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+If there's a part in a map that you struggle to get consistent on, how would you practice it to eventually pass it easily?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Although I think most players already have been trying that, actually making/editing cut difficulty and playing it (sometimes even with Half Time) would be helpful for getting consistent. However, I would never recommend to spam retry a map because there's a really huge chance to get mindblocked and I've actually seen some players suffer from mind-block for years.
```

Comment:
```suggestion
Although I think most players already have been trying that, actually making/editing cut difficulties and playing them (sometimes even with Half Time) would be helpful for getting consistent. However, I would never recommend to spam retry a map because there's a really huge chance to get mindblocked and I've actually seen some players suffer from mindblock for years.
```